### ISSUE TITLE
Figure out effects of changing the signing key

### ISSUE THREAD

**Issue by csrakowski** [csrakowski/ParallelAsync#7]
As part of updating to the new csproj format (PR: #6) and getting BenchmarkDotNet to run on `netcoreapp`, I changed the signingkey from the current pfx to a new snk one.
As this is a new key file, the resulting binaries are signed with a different public key.
I am not sure if/how this affects people who depend on the NuGet package, so it's worth looking into that before we ship out a new package with an updated key.

In short we need to find out:
* If/how this effects people
* If we need to do anything about this
* How we should proceed

**Comment by csrakowski** [comment]
Version 1.3-Preview has just been pushed to [NuGet](https://www.nuget.org/packages/CSRakowski.ParallelAsync/1.3.0-preview) in order to be tested for this.
I have updated one of my applications to use this newer version and did not see any impact at all. (The top level WIndows Service project didn't complain about anything either)

So far, it's looking good and transparent. Will try a few more things this week, and will release it properly if nothing comes up.
